File: 2-semestre/uri-online-judge/javascript/1165.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let n, x, c;

n = parseInt(lines.shift());

for (let i = 0; i < n; i++) {
    x = parseInt(lines.shift());

    c = 0;

    for (let j = 1; j <= x; j++) {
        if (x % j === 0) c++;
    }

    if (c === 2) console.log(`${x} eh primo`);
    else console.log(`${x} nao eh primo`);
}

File: 2-semestre/uri-online-judge/c/1165.c
#include <stdio.h>

int main() {
    int n, x, c, i, j;

    scanf("%d", &n);

    for (i = 0; i < n; i++) {
        scanf("%d", &x);

        c = 0;

        for (j = 1; j <= x; j++) {
            if (x % j == 0) c++;
        }

        if (c == 2) printf("%d eh primo\n", x);
        else printf("%d nao eh primo\n", x);
    }

    return 0;
}
